src/cli/cli.cpp

```diff
@@ -620,6 +620,25 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
                 otBorderAgentSetEphemeralKeyCallback(GetInstancePtr(), nullptr, nullptr);
             }
         }
+        /**
+         * @cli ba ephemeralkey feature (enable, disable)
+         * @code
+         * ba ephemeralkey feature
+         * Enabled
+         * Done
+         * ba ephemeralkey feature enable
+         * Done
```

Comment:
```suggestion
         * Done
         * @endcode
         * @code
         * ba ephemeralkey feature enable
         * Done
```

The CLI prompt only appears for the first line in a block, so we need to separate into 2 blocks here

Reply:
Thanks! Updated.